jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    preset: 'ts-jest',
```

Comment:
You may find a better option is to use the latest version of babel to transform typescript... you will still need to configure the typescript compiler to do typechecking but it will be faster and use less memory, as all it does is strip type annotations out.

Only problem is you can't use a few weird typescript only things like namespaces but you aren't using them anyway.

Reply:
That sounds amazing. I will look into it.